editor/sidebar/post-taxonomies/flat-taxonomy.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { unescape, find, throttle } from 'lodash';
```

Comment:
Note entirely in scope but: The reason I'd renamed this in the other component is because there is a global function by the same name:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape

This is problematic if the import is ever dropped for any reason, since linting will not surface any issues:

```
gutenberg (add/custom-taxonomies) $ git diff
diff --git a/editor/sidebar/post-taxonomies/flat-taxonomy.js b/editor/sidebar/post-taxonomies/flat-taxonomy.js
index 995aa17a..d21a7be6 100644
--- a/editor/sidebar/post-taxonomies/flat-taxonomy.js
+++ b/editor/sidebar/post-taxonomies/flat-taxonomy.js
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { unescape, find, throttle } from 'lodash';
+import { find, throttle } from 'lodash';
 
 /**
  * WordPress dependencies
gutenberg (add/custom-taxonomies) $ npx eslint editor/sidebar/post-taxonomies/flat-taxonomy.js 
gutenberg (add/custom-taxonomies) $ 
```